RETURN_HEADER setting added, better docs, version 1.0.0 (Closes #16) (#21)
JonasKs/django-guid: CHANGELOG.rst

=========

`1.0.0`_ - 2020-01-14
---------------------

**Features**

* Added a ``RETURN_HEADER`` setting, which will return the GUID as a header with the same name

**Improvements**

* Added a Django Rest Framework test and added DRF to the ``demoproj``

* Improved tests to also check for headers in the response

* Added tests for the new setting

> * Added examples on how the log messages gets formatted in the ``README.rst`` and in the docs

Reviewer: ```suggestion
* Added examples to ``README.rst`` and docs, to show how the log messages get formatted
```
